pkg/cvo/availableupdates.go

```diff
@@ -95,12 +108,24 @@ func (u *availableUpdates) NeedsUpdate(original *configv1.ClusterVersion) *confi
 
 // setAvailableUpdates updates the currently calculated version of updates.
 func (optr *Operator) setAvailableUpdates(u *availableUpdates) {
+	success := false
 	if u != nil {
-		u.At = time.Now()
+		u.LastAttempt = time.Now()
+		if u.Condition.Type == configv1.RetrievedUpdates {
+			success = u.Condition.Status == configv1.ConditionTrue
+		} else {
+			klog.V(4).Warnf("Unrecognized condition %s=%s (%s: %s): cannot judge update retrieval success", condition.Type, condition.Status, condition.Reason, condition.Message)
```

Comment:
Apparently `Warnf` [is not a thing][1] and I should drop `V` and use [`Warningf`][2].

[1]: https://deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gcs/origin-ci-test/pr-logs/pull/openshift_cluster-version-operator/368/pull-ci-openshift-cluster-version-operator-master-images/1032#1:build-log.txt%3A65
[2]: https://godoc.org/github.com/kubernetes/klog#Warningf

Reply:
Fixed with 8e0f5c5 -> d62aba1.